drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -301,7 +299,7 @@ def do_attempt(self, attempt, suffix=''):
 
         self.runtime.publish(self, 'edx.drag_and_drop_v2.item.dropped', {
             'item_id': item['id'],
-            'location': zone.get("title"),
+            'location': self.ugettext(zone.get("title")),
```

Comment:
@mattdrayer If this is translated, is it possible that the event logs could end up containing entries that refer to the same zone but contain different values for `location` (e.g., "first zone"/"primera zona"/"erste Zone")? If so, it might be better to default to the untranslated value here.